### DISCUSSION TITLE
节点文档汇总

### DISCUSSION THREAD

**Discussion by CavinHuang** [CavinHuang/comfyui-nodes-docs#5] [Announcements]
- [ ] AB SamplerCustom (experimental)
- [x] AddNoise
- [ ] AlignYourStepsScheduler
- [ ] BaseModel_Loader_local
- [x] BasicGuider
- [x] BasicScheduler
- [x] Blend Latents (PPF Noise)
- [x] CFGGuider
- [ ] CLIPAttentionMultiply
- [x] CLIPLoader
- [x] CLIPMergeAdd
- [x] CLIPMergeSimple
- [x] CLIPMergeSubtract
- [x] CLIPSave
- [x] CLIPSetLastLayer
- [x] CLIPTextEncode
- [x] CLIPTextEncodeControlnet
- [x] CLIPTextEncodeSDXL
- [x] CLIPTextEncodeSDXLRefiner
- [x] CLIPVisionEncode
- [x] CLIPVisionLoader
- [x] Canny
- [x] Canvas_Tab
- [x] CheckpointLoader
- [x] CheckpointLoaderSimple
- [x] CheckpointSave
- [ ] ColorizeDepthmap
- [x] ConditioningAverage
- [x] ConditioningCombine
- [x] ConditioningConcat
- [x] ConditioningSetArea
- [x] ConditioningSetAreaPercentage
- [x] ConditioningSetAreaStrength
- [x] ConditioningSetMask
- [x] ConditioningSetTimestepRange
- [x] ConditioningZeroOut
- [x] ControlNetApply
- [x] ControlNetApplyAdvanced
- [x] ControlNetLoader
- [ ] CreateRequestMetadata
- [x] CropMask
- [x] Cross-Hatch Power Fractal (PPF Noise)
- [x] Cross-Hatch Power Fractal Settings (PPF Noise)
- [x] DiffControlNetLoader
- [x] DifferentialDiffusion
- [x] DiffusersLoader
- [x] DisableNoise
- [x] DualCFGGuider
- [x] DualCLIPLoader
- [x] EmptyImage
- [x] EmptyLatentImage
- [x] ExponentialScheduler
- [x] FeatherMask
- [x] FlipSigmas
- [x] FreeU
- [x] FreeU_V2
- [x] GLIGENLoader
- [x] GLIGENTextBoxApply
- [x] GrowMask
- [x] HyperTile
- [x] HypernetworkLoader
- [ ] ICLightAppply
- [x] ImageBatch
- [x] ImageBlend
- [x] ImageBlur
- [x] ImageColorToMask
- [x] ImageCompositeMasked
- [x] ImageCrop
- [x] ImageFromBatch
- [x] ImageInvert
- [x] ImageOnlyCheckpointLoader
- [x] ImageOnlyCheckpointSave
- [x] ImagePadForOutpaint
- [x] ImageQuantize
- [x] ImageScale
- [x] ImageScaleBy
- [x] ImageScaleToTotalPixels
- [x] ImageSharpen
- [x] ImageToMask
- [x] ImageUpscaleWithModel
- [x] Images as Latents (PPF Noise)
- [x] InpaintModelConditioning
- [x] InstructPixToPixConditioning
- [x] InvertMask
- [x] JoinImageWithAlpha
- [x] KSampler
- [x] KSamplerAdvanced
- [x] KSamplerSelect
- [x] KarrasScheduler
- [x] LCMScheduler
- [x] Latent Adjustment (PPF Noise)
- [x] LatentAdd
- [x] LatentBatch
- [x] LatentBatchSeedBehavior
- [x] LatentBlend
- [x] LatentComposite
- [x] LatentCompositeMasked
- [x] LatentCrop
- [x] LatentFlip
- [x] LatentFromBatch
- [x] LatentInterpolate
- [x] LatentMultiply
- [x] LatentRotate
- [x] LatentSubtract
- [x] LatentUpscale
- [x] LatentUpscaleBy
- [x] Latents to CPU (PPF Noise)
- [x] Linear Cross-Hatch Power Fractal (PPF Noise)
- [ ] Load 64 Encoded Image
- [x] LoadImage
- [x] LoadImageMask
- [x] LoadLatent
- [ ] LoraLoader
- [x] LoraLoaderModelOnly
- [ ] MarigoldDepthEstimation
- [ ] MarigoldDepthEstimationVideo
- [x] MaskComposite
- [x] MaskToImage
- [x] ModelMergeAdd
- [x] ModelMergeBlocks
- [x] ModelMergeSD1
- [x] ModelMergeSD2
- [x] ModelMergeSDXL
- [x] ModelMergeSimple
- [x] ModelMergeSubtract
- [x] ModelSamplingContinuousEDM
- [x] ModelSamplingDiscrete
- [x] ModelSamplingStableCascade
- [x] Morphology
- [ ] NEW_PhotoMaker_Generation
- [x] PatchModelAddDownscale
- [x] Perlin Power Fractal Latent (PPF Noise)
- [x] Perlin Power Fractal Settings (PPF Noise)
- [x] PerpNeg
- [x] PerpNegGuider
- [x] PerturbedAttentionGuidance
- [ ] PhotoMakerAdapter_Loader_local
- [x] PhotoMakerEncode
- [x] PhotoMakerLoader
- [x] PolyexponentialScheduler
- [x] PorterDuffImageComposite
- [x] Power KSampler Advanced (PPF Noise)
- [x] Power-Law Noise (PPF Noise)
- [ ] Preview Chooser Fabric
- [x] PreviewImage
- [ ] Prompt_Styler
- [x] RandomNoise
- [x] RebatchImages
- [x] RebatchLatents
- [ ] Ref_Image_Preprocessing
- [ ] RemapDepth
- [x] RepeatImageBatch
- [x] RepeatLatentBatch
- [ ] RequestInputs
- [x] RescaleCFG
- [x] SDTurboScheduler
- [ ] SDXL Empty Latent Image
- [x] SDXLPromptStyler
- [x] SDXLPromptStylerAdvanced
- [x] SD_4XUpscale_Conditioning
- [x] SV3D_Conditioning
- [x] SVD_img2vid_Conditioning
- [x] SamplerCustom
- [x] SamplerCustomAdvanced
- [x] SamplerDPMAdaptative
- [x] SamplerDPMPP_2M_SDE
- [x] SamplerDPMPP_3M_SDE
- [x] SamplerDPMPP_SDE
- [x] SamplerEulerAncestral
- [x] SamplerLCMAlternative
- [x] SamplerLCMCycle
- [ ] SamplerLCMUpscale
- [x] SamplerLMS
- [ ] Save Image (api)
- [x] SaveAnimatedPNG
- [x] SaveAnimatedWEBP
- [x] SaveImage
- [ ] SaveImageOpenEXR
- [x] SaveLatent
- [x] SelfAttentionGuidance
- [x] Send_To_Editor
- [x] SetLatentNoiseMask
- [ ] SetRequestStateToComplete
- [x] SolidMask
- [x] SplitImageWithAlpha
- [x] SplitSigmas
- [ ] SplitSigmasDenoise
- [x] StableCascade_EmptyLatentImage
- [x] StableCascade_StageB_Conditioning
- [x] StableCascade_StageC_VAEEncode
- [x] StableCascade_SuperResolutionControlnet
- [x] StableZero123_Conditioning
- [x] StableZero123_Conditioning_Batched
- [ ] String to Float
- [ ] String to Integer
- [x] StyleModelApply
- [x] StyleModelLoader
- [x] ThresholdMask
- [x] TomePatchModel
- [x] UNETLoader
- [ ] UNetCrossAttentionMultiply
- [ ] UNetSelfAttentionMultiply
- [ ] UNetTemporalAttentionMultiply
- [x] UpscaleModelLoader
- [x] VAEDecode
- [x] VAEDecodeTiled
- [x] VAEEncode
- [ ] VAEEncodeArgMax
- [x] VAEEncodeForInpaint
- [x] VAEEncodeTiled
- [x] VAELoader
- [x] VAESave
- [x] VPScheduler
- [x] VideoLinearCFGGuidance
- [x] VideoTriangleCFGGuidance
- [x] unCLIPCheckpointLoader
- [x] unCLIPConditioning

**Comment by rui40000** [comment]
非常感谢你的贡献，这对于新人会很有帮助。❥(^_-)

**Reply by rui40000** [reply]
考虑到节点数量太多且源源不断，应该让用户可以自己编辑节点的说明。
不是去文件夹里编辑.md，是直接在Comfyui展开的弹板上编辑。
这样对于很多没有覆盖的冷僻节点，用户可以自己备注一下。
：）

**Reply by rui40000** [reply]
做成共建更好，节点作者自己也可以上去登记
